backend/db/setup.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Connect to the database and setup it with some default data.
+ */
+"use strict";
+
+const database = require('../db/database.js');
+
+// // This is for setting up database with data from setup.json
+// const setupFile = "setup.json";
+// const fs = require("fs");
+// const path = require("path");
+// const docs = JSON.parse(fs.readFileSync(
+//     path.resolve(__dirname, setupFile),
+//     "utf8"
+// ));
+
+// Do it.
+resetCollection("tickets")
+    .catch(err => console.log(err));
+
+/**
+ * Reset a collection by removing existing content.
+ * Can also be used to insert a default set of documents.
+ *
+ * @async
+ *
+ * @param {string} colName Name of collection.
+ * @param {string=} doc    Documents to be inserted into collection.
+ *
+ * @throws Error when database operation fails.
+ *
+ * @return {Promise<void>} Void
+ */
+async function resetCollection(colName, doc) {
```

Comment:
Parametern _doc_ ska tas bort?

Reply:
Då ska vi ju heller inte deklarera doc som du nämnde ovan?

Om du läser kommentaren till resetCollection funktionen så ser du att jag har skrivit att "Can also be used to insert a default set of documents." Som nämnt förut så är tanken att man ska kunna sätta in data via en backup-fil, men som sagt inget vi använder nu men är ju en rätt bra funktionalitet att ha i en sån typ av funktion tänker jag.

Reply:
Ah, jag ser nu att du gjorde parametern valfri 👍